docs/design/2022-03-09-optimize-gc-for-internal-transaction.md

```diff
@@ -0,0 +1,141 @@
+# Optimize GC Advance For Internal Transactions
+
+- Author(s): [Michael](https://github.com/TonsnakeLin) 
+- Tracking Issue: https://github.com/pingcap/tidb/issues/32725
+
+## Introduction
+
+This document describes the design of the feature "optimize gc advance for internal transaction", which adds the startTS(start time stamp) of all internal transactions to the gc(garbage collect) safe point calculation process.
```

Comment:
I see some "time stamp" and "safe point" usage, and I prefer "timestamp" and "safepoint".

Reply:
done